app/controllers/member/inquiry_controller.rb

```diff
@@ -5,22 +5,22 @@ def new
     @inquiry = Inquiry.new
   end
 
-  def create
-  end
-
   def confirm
     @inquiry = Inquiry.new(inquiry_params)
-    @inquiry.member_id = current_member.id
-
     render :new if @inquiry.invalid?
   end
 
+  def create
+    @inquiry = current_member.inquiries.create!(inquiry_params)
+    redirect_to action: :thanks
```

Comment:
`redirect_to action: :thanks` 
こういう書き方あまりしないです。

下記のように `xx_path` の方がいいと思います
```suggestion
    redirect_to xx_path
```

Reply:
690e8e5

書き方変更しました。